scripts/posteffects/posteffect-bloom.js

```diff
@@ -295,6 +295,6 @@ Bloom.prototype.initialize = function () {
 
     this.on('destroy', function () {
         queue.removeEffect(this.effect);
-        this._destroy();
-    });
+        this.effect._destroy();
+    }, this);
```

Comment:
I didn't think the final `this` parameter to `on` is needed here. Can you confirm?

Reply:
You are right. I just added it because had seen it in other places. Example: https://github.com/playcanvas/engine/blob/3fead3ac2a2a7bf3d09b292a07abed2b4a798cb6/scripts/posteffects/posteffect-bloom.js#L286